LibreHardwareMonitor/UI/PlotPanel.cs

```diff
@@ -158,7 +186,7 @@ private PlotModel CreatePlotModel()
             model.Axes.Add(_timeAxis);
             foreach (LinearAxis axis in _axes.Values)
                 model.Axes.Add(axis);
-            model.PlotMargins = new OxyThickness(0);
+            //model.PlotMargins = new OxyThickness(0);
```

Comment:
Can be removed if unused. Also: what would this default to now if no settings have been set?

Reply:
OK.
The ScaledPlotModel constructor sets PlotMargins.